### PR TITLE
Task.1.0

### PR DESCRIPTION
**Pull request by hungtq2016**

### CODE REVIEW

File: Start/HomeViewController.swift [hunk 1]
```diff
@@ -50,83 +41,62 @@ class HomeViewController: UIViewController, UITableViewDelegate, UITableViewData
         if entry.image != nil {
             cell.imgAppIcon.sd_setImageWithURL(NSURL(string: entry.image!), placeholderImage: UIImage(named: "no-image"),options: SDWebImageOptions.HighPriority)
         }
-        
         return cell
     }
-
+    
     func tableView(tableView: UITableView, heightForRowAtIndexPath indexPath: NSIndexPath) -> CGFloat {
         return 70
```
Comment (huynvgmo) [inline Start/HomeViewController.swift]: remove this func

File: Start/LishCategoryViewController.swift [hunk 2]
```diff
@@ -58,37 +46,19 @@ class LishCategoryViewController: UIViewController, UITableViewDataSource, UITab
         }
         return cell
     }
-    func tableView(tableView: UITableView, heightForRowAtIndexPath indexPath: NSIndexPath) -> CGFloat {
-        return 40
-    }
+    
     func tableView(tableView: UITableView, didSelectRowAtIndexPath indexPath: NSIndexPath) {
-        
         if indexPath.section == 0  {
```
Comment (huynvgmo) [inline Start/LishCategoryViewController.swift]: refactor

File: Start/HomeViewController.swift (hunk 1 above)
Comment (hungtq2016) [inline Start/HomeViewController.swift]: done

File: Start/LishCategoryViewController.swift (hunk 2 above)
Comment (hungtq2016) [inline Start/LishCategoryViewController.swift]: done

**Comment by huynvgmo** [comment]
@hungtq2016 pls merge

**Pull request merged**